scripts/runFabtests.sh

```diff
@@ -20,70 +28,79 @@ fi
 declare -r ssh="ssh -o StrictHostKeyChecking=no -o ConnectTimeout=2 -o BatchMode=yes"
 declare -r tssh="timeout ${TO} ${ssh}"
 
-output=""
-pass_count=0
-fail_count=0
+declare -r c_outp=$(mktemp)
+declare -r s_outp=$(mktemp)
+
+declare -i skip_count=0
+declare -i pass_count=0
+declare -i fail_count=0
 
 simple_tests=(
-    "cq_data"
-    "dgram"
-    "dgram_waitset"
-    "msg"
-    "msg_epoll"
-    "poll"
-    "rdm"
-    "rdm_rma_simple"
-    "rdm_shared_ctx"
-    "rdm_tagged_search"
-    "scalable_ep"
-    "cmatose"
+	"cq_data"
+	"dgram"
+	"dgram_waitset"
+	"msg"
+	"msg_epoll"
+	"poll"
+	"rdm"
+	"rdm_rma_simple"
+	"rdm_shared_ctx"
+	"rdm_tagged_search"
+	"scalable_ep"
+	"cmatose"
 )
 
 quick_tests=(
-    "msg_pingpong -I 100 -S 1024"
-    "msg_rma -o write -I 100 -S 1024"
-    "msg_rma -o read -I 100 -S 1024"
-    "msg_rma -o writedata -I 100 -S 1024"
-    "rdm_atomic -I 100 -S 1024 -o all"
-    "rdm_cntr_pingpong -I 100 -S 1024"
-    "rdm_inject_pingpong -I 100 -S 128"
-    "rdm_multi_recv -I 100 -S 1024"
-    "rdm_pingpong -I 100 -S 1024"
-    "rdm_rma -o write -I 100 -S 1024"
-    "rdm_rma -o read -I 100 -S 1024"
-    "rdm_rma -o writedata -I 100 -S 1024"
-    "rdm_tagged_pingpong -I 100 -S 1024"
-    "ud_pingpong -I 100 -S 1024"
-    "rc_pingpong -n 100 -S 1024"
+	"msg_pingpong -I 100 -S 1024"
+	"msg_rma -o write -I 100 -S 1024"
+	"msg_rma -o read -I 100 -S 1024"
+	"msg_rma -o writedata -I 100 -S 1024"
+	"rdm_atomic -I 100 -S 1024 -o all"
+	"rdm_cntr_pingpong -I 100 -S 1024"
+	"rdm_inject_pingpong -I 100 -S 128"
```

Comment:
Actually -S option was given to be consistent with other examples. But we can remove it from the script since the tests already runs for the inject_size from tx_attr by default.